src/rtctools_heat_network/pycml/component_library/heat/source.py

```diff
@@ -6,6 +6,16 @@
 
 
 class Source(_NonStorageComponent):
+    """
+    The source component is there to insert thermal power (Heat) out of the system.
+
+    The heat to discharge constraints are set in the HeatMixin. We enforce that the outgoing
+    temperature of the source matches the heat, similar as with the demands. This allows us to
+    guarantee  that the flow can always carry the heat and that thermal losses are always
```

Comment:
Same as in the demand, it is unclear what is meant by the comment: "We enforce that the outgoing temperature of the source matches ..." etc

Reply:
lets discuss